iib/workers/tasks/git_utils.py

```diff
@@ -78,18 +73,17 @@ def push_configs_to_git(
             # Configure Git user for commits
             configure_git_user(local_repo_dir)
 
-            # Copy configs/ subdirs to local Git repo
+            # Overwrite local Git repo configs/
             repo_configs_dir = os.path.join(local_repo_dir, 'configs')
+            if os.path.exists(repo_configs_dir):
+                shutil.rmtree(repo_configs_dir)
+
             log.info(
                 "Copying content of %s to local Git repository %s",
                 src_configs_path,
                 repo_configs_dir,
             )
-            for operator_package in operator_packages:
-                src_pkg_dir = os.path.join(src_configs_path, operator_package)
-                dest_pkg_dir = os.path.join(repo_configs_dir, operator_package)
-                os.makedirs(dest_pkg_dir, exist_ok=True)
-                shutil.copytree(src_pkg_dir, dest_pkg_dir, dirs_exist_ok=True)
+            shutil.copytree(src_configs_path, repo_configs_dir)
```

Comment:
Will this work if `configs` get deleted? It is, will it recreate this config dir?